SADL/AM-CDM-material.sadl

```diff
@@ -6,7 +6,7 @@ PowderLot (note "Unique lot of powder material used in one or more builds.") is
 	described by virginOrUsed (note "If the powder lot is new (virgin) or been used previously.") with a single value of type string,

```

Comment:
Should virginOrUsed be deleted? It seems useless in the context of reuseNumber.

Reply:
I agree, this information can be inferred based on reuseNumber

Reply:
To be considered in pull request #17